gradle/libs.versions.toml

```diff
@@ -27,15 +26,22 @@ kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotl
 kotlin-compiler-embeddable = { module = "org.jetbrains.kotlin:kotlin-compiler-embeddable", version.ref = "kotlin" }
 kotlin-gradle-plugin-api = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin-api", version.ref = "kotlin" }
 
+compose-ui-backhandler = "org.jetbrains.compose.ui:ui-backhandler:1.8.1"
```

Comment:
please reuse `jetbrains-compose` variable to update all compose dependencies once